src/sinks/http.rs

```diff
@@ -108,48 +110,52 @@ fn http(config: HttpSinkConfig, acker: Acker) -> Result<super::RouterSink, Strin
     let headers = config.headers.clone();
     let basic_auth = config.basic_auth.clone();
     let method = config.method.clone().unwrap_or(HttpMethod::Post);
+    let verify = config.verify_certificate.unwrap_or(true);
```

Comment:
can we add an if branch below this to spit out a warning that this setting was enabled?

Reply:
Sure. Is what I just pushed out what you are looking for?